api-operator/pkg/mgw/config.go

```diff
@@ -195,6 +258,30 @@ func SetApimConfigs(client *client.Client) error {
 	Configs.EnableRequestValidation = apimConfig.Data[enableRequestValidationConst]
 	Configs.EnableResponseValidation = apimConfig.Data[enableResponseValidationConst]
 	Configs.LogLevel = apimConfig.Data[logLevelConst]
+	Configs.EnabledAPIKeyIssuer = apimConfig.Data[enabledAPIKeyIssuerConst]
+	Configs.APIKeyKeystorePath = apimConfig.Data[apiKeyKeystorePathConst]
+	Configs.APIKeyKeystorePassword = apimConfig.Data[apiKeyKeystorePasswordConst]
+	Configs.APIKeyIssuerName = apimConfig.Data[apiKeyIssuerNameConst]
+	Configs.APIKeyIssuerCertificateAlias = apimConfig.Data[apiKeyIssuerCertificateAliasConst]
+	validityTime, err := strconv.Atoi(apimConfig.Data[validityTimeConst])
+	if err != nil {
+		logConf.Error(err, "Provided validity time is not valid. Using the default validity time")
+		Configs.ValidityTime = validityTimeValConst
+	} else {
+		Configs.ValidityTime = int32(validityTime)
+	}
+	var apiKeyTokenIssuerAPIs []APIKeyTokenAllowedAPIs
+	apiKeyIssuerAPIs := APIKeyTokenAllowedAPIs{}
+	splitArray := strings.Split(apimConfig.Data[allowedAPIsConst], "\n")
+	for _, element := range splitArray {
+		if element != "" && strings.ContainsAny(element, ":") {
+			splitValues := strings.Split(element, ":")
+			apiKeyIssuerAPIs.AllowedAPIName = strings.TrimSpace(splitValues[0])
+			apiKeyIssuerAPIs.AllowedAPIVersions = strings.TrimSpace(splitValues[1])
+			apiKeyTokenIssuerAPIs = append(apiKeyTokenIssuerAPIs, apiKeyIssuerAPIs)
+		}
+	}
```

Comment:
Or else shall we use
```go
type APIKeyTokenAllowedAPIs []map[string]string
```
instead of
```go
type APIKeyTokenAllowedAPIs struct {
	AllowedAPIName 		string
	AllowedAPIVersions 	string
}
```

Reply:
resolved